site2/docs/io-quickstart.md

```diff
@@ -386,3 +344,407 @@ bin/pulsar-admin sink delete \
+      password: "jdbc"
+      jdbcUrl: "jdbc:mysql://127.0.0.1:3306/pulsar_mysql_jdbc_sink"
+      tableName: "pulsar_mysql_jdbc_sink"
+    ```
+
+2. Create a schema.
+
+    Create a _avro-schema_ file, copy the following contents to this file, and place the file in the `pulsar/connectors` folder.
+
+    ```text
+    {
+      "type": "AVRO",
+      "schema": "{\"type\":\"record\",\"name\":\"Test\",\"fields\":[{\"name\":\"id\",\"type\":[\"null\",\"int\"]},{\"name\":\"name\",\"type\":[\"null\",\"string\"]}]}",
+      "properties": {}
+    }
+    ```
+
+    > #### Tip
+    >
+    > For more information about AVRO, see [Apache Avro Documentation](https://avro.apache.org/docs/1.8.2/).
+
+
+3. Upload a schema to a topic.  
+
+    This example uploads the _avro-schema_ schema to the _pulsar-mysql-jdbc-sink-topic_ topic.
+
+    ```text
+    $ bin/pulsar-admin schemas upload pulsar-mysql-jdbc-sink-topic -f ./connectors/avro-schema
+    ```
+
+4. Check if the schema has been uploaded successfully.
+
+    ```text
+    $ bin/pulsar-admin schemas get pulsar-mysql-jdbc-sink-topic
+    ```
+
+    The schema has been uploaded successfully if the following message appears.
+
+    ```text
+    {
+      "name" : "pulsar-mysql-jdbc-sink-topic",
+      "schema" : "eyJ0eXBlIjoicmVjb3JkIiwibmFtZSI6IlRlc3QiLCJmaWVsZHMiOlt7Im5hbWUiOiJpZCIsInR5cGUiOlsibnVsbCIsImludCJdfSx7Im5hbWUiOiJuYW1lIiwidHlwZSI6WyJudWxsIiwic3RyaW5nIl19XX0=",
+      "type" : "AVRO",
+      "properties" : { }
+    }
+    ```
+
+### Submit a JDBC sink
+
+Pulsar provides the [CLI](admin-api-overview.md) for running and managing Pulsar I/O connectors.
+
+This example creates a sink connector and specifies the desired information.
+
+```text
+$ bin/pulsar-admin sink create \
+--archive ./connectors/pulsar-io-jdbc-{{pulsar:version}}.nar \
+--inputs pulsar-mysql-jdbc-sink-topic \
+--name pulsar-mysql-jdbc-sink \
+--sink-config-file ./connectors/pulsar-mysql-jdbc-sink.yaml \
+--parallelism 1
+```
+
+Once the command is executed, Pulsar will create a sink connector named _pulsar-mysql-jdbc-sink_, and the sink connector will be running as a Pulsar Function and write the messages produced in the  _pulsar-mysql-jdbc-sink-topic_ topic to the MySQL _pulsar_mysql_jdbc_sink_ table.
+
+> #### Tip
+>
+> Flag | Description | This example
+> - | - | - |
+> `--archive` | Path to the archive file for the sink. | _pulsar-io-jdbc-{{pulsar:version}}.nar_ 
+> `--inputs` | The input topic or topics of the sink. <br> (Multiple topics can be specified as a comma-separated list.)
+> `--name` | The name of the sink. | _pulsar-mysql-jdbc-sink_
+> `--sink-config-file` | The path to a YAML config file specifying the configuration of the sink. | _pulsar-mysql-jdbc-sink.yaml_ 
+> `--parallelism` | The parallelism factor of the sink. <br> For example, the number of sink instances to run. |  _1_
+> For more information about `pulsar-admin sink create options`, see [here](https://pulsar.apache.org/docs/en/pulsar-admin/#create-3).
```

Comment:
@Anonymitaet same here. I think we should try to figure out if we can use relative url.

`[here](pulsar-admin/#create-3)`

Reply:
Thanks for reminding, now I understand # is an anchor pointing to a place within a page, this is what I want.